### DISCUSSION TITLE
v0.3.1

### DISCUSSION THREAD

**Discussion by mxcl** [teaxyz/teaBASE#10] [Announcements]
* be aware of alternative git config locations, eg. ~/.config/git/config
* fix crash if no git identity set

**Full Changelog**: https://github.com/teaxyz/teaBASE/compare/v0.3.0...v0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/teaxyz/teaBASE/releases/tag/v0.3.1'>v0.3.1</a>.</em>

**Comment by Ayaan1982** [comment]
Great 👍
